### PR TITLE
feat: add language select box and background color change when hover

### PR DESCRIPTION
**Pull request by turbodog03**
## changes：
* Removed the origin language select box on footer, and add a new one on header. #165 
* Add shadow for right dropdown when hover 

## preview:
http://39.106.18.82/

### CODE REVIEW

**Comment by turbodog03** [comment]
@nodece  Please review

File: web/src/muti_language.svg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg t="1626179895478" class="icon" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" p-id="1276" xmlns:xlink="http://www.w3.org/1999/xlink" width="200" height="200"><defs><style type="text/css">html, * { user-select: text !important; }
```
Comment (hsluoyz) [inline web/src/muti_language.svg:1]: Static resources like SVG should be put to static: https://github.com/casbin/static/tree/master/img , then refer to to with URL

File: web/src/App.less [hunk 2]
```diff
@@ -39,3 +39,21 @@
...
+  background-size: 25px, 25px;
+  width: 25px;
+  height: 25px;
+  margin: 22px 20px 16px 20px;
+  float: right;
+  cursor: pointer;
+  &:hover {
+    background-color: #f5f5f5;
+  }
+}
```
Comment (hsluoyz) [inline web/src/App.less:54]: new line

File: web/src/SelectLanguageBox.js [hunk 3]
```diff
@@ -14,7 +14,12 @@
 
 import React from "react";
 import * as Setting from "./Setting";
-
+import { Menu, Dropdown, message } from "antd";
+import { createFromIconfontCN } from '@ant-design/icons';
+import './App.less';
+const IconFont = createFromIconfontCN({
```
Comment (hsluoyz) [inline web/src/SelectLanguageBox.js:20]: new line

File: web/src/SelectLanguageBox.js [hunk 4]
```diff
@@ -14,7 +14,12 @@
...
+  scriptUrl: '//at.alicdn.com/t/font_2680620_ffij16fkwdg.js',
+});
 class SelectLanguageBox extends React.Component {
```
Comment (hsluoyz) [inline web/src/SelectLanguageBox.js:23]: new line

File: web/src/App.js [hunk 5]
```diff
@@ -418,7 +419,7 @@ class App extends Component {
           textAlign: 'center',
         }
       }>
-        <SelectLanguageBox/>
+
```
Comment (hsluoyz) [inline web/src/App.js:422]: no new line

**Review by hsluoyz — CHANGES_REQUESTED** [review]
This time looks good! A few comments here:

image

File: web/src/muti_language.svg (hunk 1 above)
Comment (hsluoyz) [reply web/src/muti_language.svg:1]: @turbodog03 merged: https://github.com/casbin/static/pull/22

**Review by hsluoyz — APPROVED** [review]

**Comment by hsluoyz** [comment]
@ErikQQY can you take a look at why the translation doesn't work?

Demo site: http://39.106.18.82/

image

**Pull request merged**